workflows/reference/inputs/bwa-db-build-inputs.json

```
@@ -0,0 +1,7 @@
+{
+    "bwa_db_build.reference_fa_url": "https://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000/001/405/GCA_000001405.15_GRCh38/seqs_for_alignment_pipelines.ucsc_ids/GCA_000001405.15_GRCh38_no_alt_analysis_set.fna.gz",
+    "bwa_db_build.reference_fa_name": "GRCh38_no_alt.fa.gz",
+    "bwa_db_build.reference_fa_md5": "a08035b6a6e31780e96a34008ff21bd6",
+    "bwa_db_build.build_bwa_db.bwadb_dir_name": "GRCh38_no_alt.bwadb",
```

```suggestion
    "bwa_db_build.build_bwa_db.bwadb_dir_name": "GRCh38_no_alt.bwa_db",
```
Not necessary, but I think this improves readability.